mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -598,6 +602,11 @@ text_viewer.tab_size = Tab size
 text_viewer.whitespace_visible = Whitespace visible
 image_viewer.view_menu = $[text_viewer.view]
 image_viewer.controls_menu = Controls
+image_viewer.view_status_bar = Status Bar
+image_viewer.initial_zoom_menu = Initial Zoom
+image_viewer.initial_zoom_as_native = Native Size
+image_viewer.initial_zoom_fit_to_view = Fit to View
+image_viewer.initial_zoom_resize_window = Resize Window
```

Comment:
besides 'Select All' in all other properties I see only the first word with a capital letter - let's keep that convention?